loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -35,6 +35,7 @@
 
 /* App binary information for update test */
 #define APP_NAME                   "micom"
+#define MYAPP_NAME                 "wifi"
```

Comment:
I think `MICOM_APP_NAME` and `WIFI_APP_NAME` are better.

Reply:
How about `app1` and `app2`? `micom` and `wifi` are too specific.